### DISCUSSION TITLE
QFieldCloud generating duplicates of basemaps every time changes are pushed

### DISCUSSION THREAD

**Discussion by jpnunn** [opengisch/QField#4683] [Bugs]
Hi,

We are having real issues with QFieldCloud (from an organization account)  in that every time someone pushes changes, or changes are re-applied we are experiencing duplicates of basemaps appearing.  These basemap could either be a simple layer in a QGIS project as a tif or a raster in a gpkg.  I have also experienced the same issue when I have treated them as a basemap in QFieldSync.

It means that our total hard drives rapidly fill up, which then prevents everything from working.  

![image](https://github.com/opengisch/QField/assets/94465162/c0c11b11-2ab9-43aa-a688-5cfb2930c5c2)

Thanks

Joe

**Comment by BoswachterMarc** [comment]
I think these are normal backup files, no? In settings you can change number of files to keep.
![afbeelding](https://github.com/opengisch/QField/assets/57868975/6c3e033f-1b3a-4180-932d-28adc880b7d3)
